app/controllers/wrangling_guidelines_controller.rb

```diff
@@ -29,47 +28,52 @@ def manage
 
   # POST /wrangling_guidelines
   def create
-    @wrangling_guideline = WranglingGuideline.new(params[:wrangling_guideline])
+    @wrangling_guideline = WranglingGuideline.new(wrangling_guideline_params)
 
     if @wrangling_guideline.save
       flash[:notice] = ts('Wrangling Guideline was successfully created.')
       redirect_to(@wrangling_guideline)
     else
-      render :action => 'new'
+      render action: 'new'
     end
   end
 
   # PUT /wrangling_guidelines/1
   def update
-    @wrangling_guideline = WranglingGuideline.find(params[:id])
+    @wrangling_guideline = WranglingGuideline.find(wrangling_guideline_id)
 
-    if @wrangling_guideline.update_attributes(params[:wrangling_guideline])
+    if @wrangling_guideline.update_attributes(wrangling_guideline_params)
       flash[:notice] = ts('Wrangling Guideline was successfully updated.')
       redirect_to(@wrangling_guideline)
     else
-      render :action => 'edit'
+      render action: 'edit'
     end
   end
 
   # reorder FAQs
   def update_positions
     if params[:wrangling_guidelines]
-      @wrangling_guidelines = WranglingGuideline.reorder(params[:wrangling_guidelines])       
+      @wrangling_guidelines = WranglingGuideline.reorder(params.permit(:utf8, :commit, wrangling_guidelines: [])[:wrangling_guidelines])
```

Comment:
right, but the point is you ARE permitting these params. If you don't want them, don't permit them. I don't think it's counter-intuitive at all, that's the whole idea -- I mean, you could just permit! everything, and it would all work instantly, but you'd be breaking the point of the gem/Rails change, which is that it is a bad idea to permit attributes unless you explicitly want and need them. utf8 and commit are just housekeeping params that are not required past this point so deleting them seems to me the best thing to do. 

And again, I think in this method, even within wrangling_guidelines we should only be permitting position information. We wouldn't want someone submitting new text to multiple wrangling guidelines here, right?